cumulusci/utils/xml/metadata_tree.py

```diff
@@ -37,15 +38,15 @@ def parse(source):
     """
     if hasattr(source, "open"):  # for pathlib.Path objects
         with source.open() as stream:
-            doc = etree.parse(stream)
+            doc = lxml_parse_file(stream)
```

Comment:
the type annotations for lxml_parse_file say it accepts a str or a Path; looks like that should also include some sort of file-like?

Reply:
Updated and tested.

Reply:
b105bd070